src/core_ocean/shared/mpas_ocn_equation_of_state_linear.F

```diff
@@ -317,11 +320,18 @@ subroutine ocn_equation_of_state_linear_density_exp(           &
                                (tracersSurfaceLayerValue(indexS,iCell) - &
                                 ocnEqStateLinearSref)
 
-            thermalExpansionCoeff(k,iCell) = ocnEqStateLinearAlpha/ &
-                                             density(k,iCell)
-
-            salineContractionCoeff(k,iCell) = ocnEqStateLinearBeta/ &
-                                              density(k,iCell)
+            !mrp replace this ********** temporary
+!            thermalExpansionCoeff(k,iCell) = ocnEqStateLinearAlpha/ &
+!                                             density(k,iCell)
+!
+!            salineContractionCoeff(k,iCell) = ocnEqStateLinearBeta/ &
+!                                              density(k,iCell)
+            thermalExpansionCoeff(k,iCell) = ocnEqStateLinearAlpha
+            salineContractionCoeff(k,iCell) = ocnEqStateLinearBeta
+            ! mrp printing debug loop. Remove later
+            !if (iCell==6) then
+            !    print *, k,density(k,iCell),thermalExpansionCoeff(k,iCell),salineContractionCoeff(k,iCell)
+            !endif
```

Comment:
Again, I would remove the temporary stuff before merging

Reply:
Yes. Reverting all changes in this file.